### PR TITLE
[IMP] accounting: add extra taxes

### PR DESCRIPTION
**Pull request by toaa-odoo**
Extra taxes (ecotax, luxury, etc) are currently missing from doc

Forward to master.

taskid-3794931

### CODE REVIEW

File: content/applications/finance/accounting/taxes.rst [hunk 1]
```diff
@@ -307,6 +307,31 @@ directly to a product line.
...
+Extra taxes
+===========
+
+"Extra taxes" is a broad term referring to additional taxes beyond the standard or basic taxes
+imposed by governments. These extra taxes can be **luxury** taxes, **environmental** taxes,
+**import** or **export duties** taxes, etc.
+
+.. note::
+   The way luxury taxes, environmental taxes, etc., are computed varies from country to country,
+   check your country's regulations to know how to compute them for your business.
```
Comment (Donapi) [inline content/applications/finance/accounting/taxes.rst:319]:
```suggestion
   The method of computing these taxes varies across different countries. We recommend consulting your country's regulations to understand how to calculate them for your business.

File: content/applications/finance/accounting/taxes.rst [hunk 2]
```diff
@@ -307,6 +307,31 @@ directly to a product line.
    .. image:: taxes/list-sequence.png
       :alt: The taxes' sequence in Odoo determines which tax is applied first
 
+Extra taxes
+===========
+
+"Extra taxes" is a broad term referring to additional taxes beyond the standard or basic taxes
+imposed by governments. These extra taxes can be **luxury** taxes, **environmental** taxes,
```
Comment (Donapi) [inline content/applications/finance/accounting/taxes.rst:314]: Below, you speak about ecotax; is it another name for the environmental tax? Should we always use the same name to refer to this tax?

File: content/applications/finance/accounting/taxes.rst [hunk 3]
```diff
@@ -307,6 +307,31 @@ directly to a product line.
...
+**import** or **export duties** taxes, etc.
+
+.. note::
+   The way luxury taxes, environmental taxes, etc., are computed varies from country to country,
+   check your country's regulations to know how to compute them for your business.
+
+In Belgium, the formula to compute an environmental tax is: `(product price + ecotax) x sales tax`.
+Therefore, our ecotax has to come *before* the sales tax in the computation sequence.
```
Comment (Donapi) [inline content/applications/finance/accounting/taxes.rst:322]: I understand what you mean by *before*, but I'm not sure how it is explained is 100% clear... we can discuss IRL

Comment (Donapi) [reply content/applications/finance/accounting/taxes.rst:322]: Would it makes sens to move this paragraph below (or even in the example section?): 
First, we explain how to do it in Odoo, in general, and then we give an example of how we do it in Belgium?

File: content/applications/finance/accounting/taxes.rst [hunk 4]
```diff
@@ -307,6 +307,31 @@ directly to a product line.
...
+
+To do so, :ref:`create a tax <taxes/configuration>`, enter a tax name, select a
```
Comment (Donapi) [inline content/applications/finance/accounting/taxes.rst:324]:
```suggestion
To compute an extra tax in Odoo, :ref:`create a tax <taxes/configuration>`, enter a tax name, select a
```

**Review by Donapi — APPROVED** [review]
Excellent work, son!

File: content/applications/finance/accounting/taxes.rst [hunk 5]
```diff
@@ -307,6 +307,32 @@ directly to a product line.
...
+**import** or **export duties** taxes, etc.
+
+.. note::
+   The method to compute these taxes varies across different countries. We recommend consulting your
+   country's regulations to understand how to calculate them for your business.
+
+To compute an extra tax in Odoo, :ref:`create a tax <taxes/configuration>`, enter a tax name, select
+a :ref:`Tax Computation <taxes/configuration>`, set an :guilabel:`Amount`, and in the
+:guilabel:`Advanced Options` tab, check :guilabel:`Affect Base of Subsequent Taxes`. Once created,
+order the tax according to the computation order of your country.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/taxes.rst:324]: This last sentence is super important but not clear enough, in my opinion.

It's worth explaining that the sequence order of the taxes as they are listed in the taxes list view is what is going to define the computation order. The example below then reinforces super well the info.

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
A small improvement, then it will be good to go.

Thanks!

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**